### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by Sonny-Crockett** [azariah001/Star-Wars-RPG-Character-Sheet#1]
Do we need to host this? How do we import characters from OggDudes software?

**Comment by azariah001** [comment]
This is barely prealpha at this stage. I got caught up with work and stuff so I haven't had a chance to work on it.

It's configured to run as a node.js server but all of the heavy lifting is currently done by the Google Polymer based client side. You have to convert the XML character sheet exported by OggDudes Windows Application into a JSON file which is the `zeroki.json` file under the `/views/` path currently. Eventually, there'd need to be an interface for managing multiple character sheets and uploading/downloading them.

I got stuck at the point of displaying and calculating the specialisations because they don't have a consistent grid structure. There's a bunch of different ways the specialisations are connected and... the way the XML/JSON is structured it's pretty hard to interpret how things are connected.

In a nutshell unless you're a programmer willing to contribute some time, this probably isn't what you're looking for. 😞 I and a friend have actually been looking for a side project to work on recently and I'd forgotten this even existed so thanks for reminding me. 😄

**Comment by Sonny-Crockett** [comment]
You are absolutely correct, I am not programmer haha.  I am trying to track down as many resources for FFG SW RPG.  I know the guy that runs SWSheets is looking for a pull regarding an import for OggDude, and his site already handles the multiples character sheets, etc.  

https://github.com/swsheets/swsheets/issues/15

http://swsheets.com/c

Might be something to consider, teaming up lol :) 

If there is any testing or any other non-programming way I can help just let me know.
